### ISSUE TITLE
Saving WorkBook with button and macro unlinks macro from button

### ISSUE THREAD

**Issue by UFuz** [ClosedXML/ClosedXML#1203]
## Read and complete the full issue template

**Do you want to request a *feature* or report a *bug*?**
- [] Bug
- [X] Feature

**Version of ClosedXML**

0.94.2

**What is the current behavior?**

After saving a XLSM with button and macro, the link of the button to the macro gets lost.

**What is the expected behavior or new feature?**

The connection of the macro to the button should remain vaild.

**Did this work in previous versions of our tool?  Which versions?**

Not sure.

## Reproducibility

1) open excel with Macro/Button
2) do some changes
3) Save and close excel with Macro/Button

When opened in Excel application, the button "Toggle Rowfilter" can not be clicked as the link to the macro is lost.

**Code to reproduce problem:**
```c#
using (var xlWorkBook = new XLWorkbook(FileName, XLEventTracking.Disabled))
{
    IXLWorksheet xlWorkSheet = xlWorkBook.Worksheets.FirstOrDefault();
    IXLRange range = xlWorkSheet.RangeUsed();
    // do changes in WorkSheet
    xlWorkBook.Save();
}
```
- [X] I attached a sample spreadsheet.  (You can drag files on to this issue)
[Button.zip](https://github.com/ClosedXML/ClosedXML/files/3285925/Button.zip)

**Comment by igitur** [comment]
Please attach your spreadsheet as a file. The link has expired.

**Comment by UFuz** [comment]
updated link: https://abb-my.sharepoint.com/:u:/p/ulrich_fuhrmann_de/EZ2WMBB2IZJGka3R8OhiSPoBDsgKeSDt8khpBBNi9QhMzg?e=b5ee4O

**Comment by UFuz** [comment]
Possibly fixed with #1157.

**Comment by igitur** [comment]
**ATTACH YOUR SPREADSHEET ON THIS ISSUE**

**Comment by UFuz** [comment]
I REALLY TRIED THAT FROM THE BEGINNING. But it does not work. Tried IE, Chrome and FireFox.
" Something went really wrong, and we can’t process that file. " ZIP file, cause XLSM is not allowed.

**Comment by igitur** [comment]
So zip the file and attach it then.

**Comment by UFuz** [comment]
OK different computer and different browser did the trick. What a mess....

**Comment by igitur** [comment]
> OK different computer and different browser did the trick. What a mess....
Thanks

> Possibly fixed with #1157
The fix for #1157 is possibly superceded by #1227
